doc/2/guides/getting-started/run-kuzzle/index.md

```diff
@@ -32,7 +32,7 @@ You can install Kourou globally by using NPM: `npm install -g kourou`
 
 ::: warning
 Kuzzle uses compiled C++ dependencies so a compile toolchain (a C++ compiler like g++ or clang, make and python) is necessary to run `npm install kuzzle`.  
-For the sake of simplicity we will use a Docker and Docker Compose throughout this guide.  
+For the sake of simplicity we will use a Docker and Docker Compose throughout this guide. Make sure to have them correctly installed before going further, especially the [Docker as a non root user section](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user) which is a recurrent trouble.
```

Comment:
```suggestion
For the sake of simplicity we will use Docker and Docker Compose throughout this guide. Make sure to have them correctly installed before going further, especially the [Docker as a non root user section](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user) which is a potential vulnerability.
```

Reply:
I'm not sure "vulnerability" is the right word for that, it's more like people often can't run a Kuzzle for the first time because they try to use is as a non root user... Maybe something more like:
```suggestion
For the sake of simplicity we will use Docker and Docker Compose throughout this guide. Make sure to have them correctly installed before going further, especially the [Docker as a non root user section](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user) which is a potential issue.
```

Reply:
I personnaly like the way @scottinet proposed it 👍